sample/machineclass.yaml

```diff
@@ -0,0 +1,40 @@
+# Sample machine-class object
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineClass
+metadata:
+  name: small
+  namespace: foo
+capacity:
+  cpu: 2
+  memory: 7.5Gi
+  storage: 20Gi
+allocatable:
+  cpu: 2
+  memory: 7.5Gi
+  storage: 20Gi
+providerConfig:
+  apiVersion: "gceproviderconfig/v1alpha1"
+  kind: "GCEProviderConfig"
+  project: "$GCLOUD_PROJECT"
+  zone: "${ZONE:-us-central1-f}"
```

Comment:
since we are going to remove the parameters part, make this just `us-central1-f` instead of my bash-style substitution syntax.

Reply:
Fixed.